.editorconfig

```diff
@@ -1,16 +1,20 @@
-# EditorConfig: https://EditorConfig.org
+# /.editorconfig
+# Tells most editors what our style preferences are
+# https://editorconfig.org
 
-# top-most EditorConfig file
 root = true
 
-# Unix-style newlines with a newline ending every file
 [*]
+charset = utf-8
 end_of_line = lf
 insert_final_newline = true
+trim_trailing_whitespace = true
+max_line_length = 80
+indent_size = tab
+tab_width = 2
 
-# Matches multiple files with brace expansion notation
-# Set default charset
-[*.{js},{package.json,.travis.yml}]
-indent_style = space
-indent_size = 2
```

Comment:
I think we want this for `package.json`, at least

Reply:
Done!